tests/test_community.py

```diff
@@ -142,7 +140,7 @@ def test_community_put(client):
             follow_redirects=True,
         )
         assert (
-            b'{"access_token":"AccessToken","data":{"name":"TestCommunity"},"message":"Community Updated","refresh_token":"RefreshToken","status_code":200}\n'
+            b'{"access_token":"AccessToken","data":{"Admins":["Testemail@gmail.com"],"name":"TestCommunity"},"message":"Community Updated","refresh_token":"RefreshToken","status_code":200}\n'
```

Comment:
is it possible to add a second testcase that validates that when a non admin updates it fails?

Reply:
Yep! Added in new commit.